migrate-binding-to-dotnetiot.md

```diff
@@ -0,0 +1,147 @@
+# Migrate binding to .NET IoT
+
+So you have created your shiney .NET nanoFramework binding for a cool sensor or device. **That's great!** But did you know that with minimal efforts you can make your hard word available for even more .NET developers? This article describes the steps you can take to port a .NET nanoFramework binding to [.NET IoT](https://github.com/dotnet/iot).
+
+## Steps
+
+### 1. Fork the .NET IoT repo
+
+Since we can't push directly to the .NET IoT repo, we need to create a fork. This can easily be done from the GitHub web UI, or by the GitHub CLI (see the [instructions on in the GitHub docs](https://docs.github.com/en/get-started/quickstart/fork-a-repo)).
+
+### 2. Clone and build the forked repo
+
+Once we have created the fork, we can clone the forked repo. (update the URL in the command below, to match the URL of the forked repo):
+
+```shell
+git clone https://github.com/YOUR-GITHUB-ACCOUNT/iot.git
+```
+
+It's recommended to also create a branch for the work you are going to do:
+
+```shell
+git checkout -b BRANCHNAME
+```
+
+To verify your development environment has the correct .NET SDK's, and all dependencies are available locally it's recommneded to try to build the code. You can do that by running `Build.cmd` (on Windows) or `build.sh` (on Linux) which can be found in the root of the code base.
+
+### 3. Create a new project for your binding and copy code
+
+In the `/src/devices` folder, create a new subfolder for your binding (e.g. by using the name of the sensor).
+
+In this folder , create a new empty .NET project. This can be done with the Visual Studio UI, or by using the command line:
+
+```shell
+dotnet new classlib
```

Comment:
does this create a sln file as well?

Reply:
Good catch, Visual Studio will create the .sln (by default), but `dotnet new` doesn't. I'll adjust the next step/comment to fix.